core/spec/lib/spree/core/testing_support/factories/order_factory_spec.rb

```diff
@@ -45,4 +45,10 @@
 
     it_behaves_like 'a working factory'
   end
+
+  describe 'completed order with promotion' do
+    let(:factory) { :completed_order_with_promotion }
+
+    it_behaves_like 'a working factory'
+  end
```

Comment:
We seem to have a convention of not putting factories under the same level of testing as other parts of the system. Is that an intentional choice? If we were fixing any other regression outside of factories, wouldn't we be adding more robust tests?